lib/msf/ui/console/command_dispatcher/developer.rb

```diff
@@ -51,6 +63,19 @@ def reload_file(path)
     load path
   end
 
+  def reload_diff_files
+    unless git?
+      return
+    end
+    output, status = Open3.capture2e *%w(git diff --name-only)
```

Comment:
This will list ALL changed files, including non-reloadable files and non-Ruby files. Are you sure you want to fall back on `reload_file`'s error handling?

Reply:
(I'd err on the side of being more verbose while tampering with a running Metasploit instance.)